src/config.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+import os.path
+import re
+import secrets
+import string
+
+import jsonschema
+from src.ssm import get_parameter
+
+LOGGER = logging.getLogger("root")
+
+
+def schema_dir():
+    return os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + "/schema/"
+
+
+def is_invalid(data):
```

Comment:
It is easier for me to think in terms of positive conditions, so I'd do a `def is_valid(data)` instead but maybe that's just me :) Feel free to ignore